src/core_functions/scalar/string/to_base.cpp

```diff
@@ -0,0 +1,79 @@
+#include "duckdb/core_functions/scalar/string_functions.hpp"
+#include "duckdb/common/vector_operations/vector_operations.hpp"
+
+namespace duckdb {
+
+static const char alphabet[] = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+static void ToBaseFunction(DataChunk &args, ExpressionState &state, Vector &result) {
+	auto &input = args.data[0];
+	auto &radix = args.data[1];
+	auto count = args.size();
+
+	BinaryExecutor::Execute<int64_t, int32_t, string_t>(input, radix, result, count, [&](int64_t input, int32_t radix) {
+		if (input < 0) {
+			throw InvalidInputException("'to_base' number must be greater than or equal to 0");
+		}
+		if (radix < 2 || radix > 36) {
+			throw InvalidInputException("'to_base' radix must be between 2 and 36");
+		}
+
+		char buf[64];
```

Comment:
Can the code here be unified with `ToBaseFunctionWithPadding`? (it looks to be the same except `minimum_length = 0`)

Reply:
Fixed